app/views/space_contacts/_new.html.erb

```diff
@@ -27,7 +28,8 @@
                   <%= inline_svg_tag 'contact' %>
                   <%= t('space_contacts.submit') %>
                 </button>
-                <button type="button"
+                <button id="space_contacts_new"
```

Comment:
This is just a button to hide, should probably have an id other than space_contacts_new. E.g. "space_contacts_hide_toggle" (if needed for tests)